docs/source/examplecleanup.rst

```diff
@@ -6,55 +6,94 @@ End-to-End Example
+
+   This project uses [Poetry] to manage its dependencies and do a lot of the heavy lifting. So you'll need to clone the tartufo repo and setup poetry!
+
+
+   .. code-block:: console
+
+      git clone git@github.com:godaddy/tartufo.git
+      # use a virtual environment!
+      python3.7 -m venv .venv
+      . .venv/bin/activate
+      
+
+   Next: [Install Poetry]  
+
+   Finally: Run ``poetry install``
+
+
+#. Clone your repo!
+
+   Select and clone the repo you want to run tartufo on
 
    .. code-block:: console
 
-      # Run Tartufo on your repo:
+      # Clone your repo, variables used later:
       GITHUBPROJECT="yourproject"
-      GITHUBREPO="yourrepo.git"
+      GITHUBREPO="myrepo.git"
       GITHUBADDRESS="github.com"
-      tartufo git@${GITHUBADDRESS}:${GITHUBPROJECT}/${GITHUBREPO}
-      # this will spit out a bunch of stuff and end with something like: Results have been saved in /tmp/tmp_jm3hyne
-      TARTUFOSCANPATH="/tmp/tmp_jm3hyne"
-      # this is the number of individual bad password hits tartufo has found:
-      ls ${TARTUFOSCANPATH} | wc -l
+      git clone --mirror git@${GITHUBADDRESS}:${GITHUBPROJECT}/${GITHUBREPO}
 
 
-#. Take results and create a "bad password" file. 
+#. Use ``tartufo`` to scan your repository and find any secrets in its history!
 
-   This file will be used by ``BFG`` and replace these flagged "bad password" entries with ``***REMOVED***``. It is important that you read through this file to make sure there are not exceptions that you want to remove and exclude with tartufo!
+   Use what you've learned to scan your repo!
 
    .. code-block:: console
 
-      # Create a "Bad password" file
-      BADSTRINGARR=`cat ${TARTUFOSCANPATH}/* | jq .strings_found | grep -v '\[' | grep -v '\]'`
-      readarray -t PEWPEWPEWARR <<<"$BADSTRINGARR"
-      # shows entires in array, may be larger than the earlier word count
-      echo ${#PEWPEWPEWARR[@]};
-      SORTED_UNIQUE_ENTRIES=($(echo "${PEWPEWPEWARR[@]}" | tr ' ' '\n' | sort -u | tr '\n' ' '))
-      echo ${#SORTED_UNIQUE_ENTRIES[@]};
-      printf "%s\n" "${SORTED_UNIQUE_ENTRIES[@]}" > badpwz.txt
-      # remove leading/trailing quotes and trailing commas
-      sed -i -e 's/\,$//g' -e 's/\"$//g' -e 's/^\"//g' badpwz.txt
-      
-   Now you have a "bad password" file! Take a look through it, see if anything is wrong, these values will be replaced in your code history
+      # Run Tartufo on your repo and create a list of high entropy items to remove:
+      poetry run tartufo --regex --json --cleanup ${GITHUBREPO} | jq -r '.found_issues[].matched_string' | sort -u > remove.txt
 
+   Now you have a "bad password" file! Take a look through it, see if anything is wrong. This file will be used by ``BFG`` and replace these flagged "bad password" entries with ``***REMOVED***``. It is important that you read through this file to make sure there are not exceptions that you want to remove and exclude with tartufo!
```

Comment:
It would be helpful to link to the "Limiting Scans by Signature" and/or "Limiting Scans by Path" in the features doc here. That said, I looked at 2-3 other pages before I found where that was documented, and I wrote the freaking thing! So... I'm thinking the docs need a bit more of a wholesale update so that they flow better. But that's far beyond the scope of just fixing up this doc...

Reply:
Agreed, adding